python/cugraph/dask/link_analysis/pagerank.py

```diff
@@ -141,21 +138,36 @@ def pagerank(input_graph,
         if input_graph.renumbered is True:
             personalization = input_graph.add_internal_vertex_id(
                 personalization, "vertex", "vertex"
-            ).compute()
-
-    result = [client.submit(call_pagerank,
-                            Comms.get_session_id(),
-                            wf[1],
-                            num_verts,
-                            num_edges,
-                            vertex_partition_offsets,
-                            alpha,
-                            max_iter,
-                            tol,
-                            personalization,
-                            nstart,
-                            workers=[wf[0]])
-              for idx, wf in enumerate(data.worker_to_parts.items())]
+            )
+        p_data = get_distributed_data(personalization)
+
+        result = [client.submit(call_pagerank,
+                                Comms.get_session_id(),
+                                wf[1],
+                                num_verts,
+                                num_edges,
+                                vertex_partition_offsets,
+                                alpha,
+                                max_iter,
+                                tol,
```

Comment:
I think we need to discuss more about ```tol```.

The code below is what NetworkX does (compare ```err``` with ```N * tol```). This requires setting tol to a smaller value if N gets large (tol here basically means tolerance for a single PageRank value not the entire set of PageRank vector).

https://github.com/networkx/networkx/blob/master/networkx/algorithms/link_analysis/pagerank_alg.py#L155
```
err = sum([abs(x[n] - xlast[n]) for n in x])
        if err < N * tol:
            return x
```

The new PageRank algorithm adopts this NetworkX logic to determine convergence, so should we follow the NetworkX logic (and set tol to a smaller value with a larger N) or better remove ```N * ``` from the new PageRank code?